src/getter.rs

```diff
@@ -509,4 +509,17 @@ impl Getter for CppCode {
 
 impl Getter for PreprocCode {}
 impl Getter for CcommentCode {}
-impl Getter for JavaCode {}
+impl Getter for JavaCode {
+    fn get_space_kind(node: &Node) -> SpaceKind {
+        use Java::*;
+
+        let typ = node.object().kind_id();
+        match typ.into() {
+            MethodDeclaration | LambdaExpression => SpaceKind::Function,
+            Class | ClassDeclaration => SpaceKind::Class,
```

Comment:
Did you intentionally miss interface:
https://github.com/tree-sitter/tree-sitter-java/blob/a24ae7d16de3517bff243a87d087d0b4877a65c5/grammar.js#L632
?
If yes, please add a comment to explain why.
In looking at the above link I can see that there is a `module` concept (I don't know what is: I didn't write Java code for years), so same question.

Reply:
Good catch. Since this is my first time working in Rust, this repo, and tree-sitter, rather than copying other implementations I just added grammar to make tests pass. 
I will take a scan through to see if any other things I missed.
It has been a decade or so since I have looked at Java... I do not remember ever using `module` :D I learned something today.

Reply:
For the SLOC off by 1 issue the only match I need is `Program => SpaceKind::Unit`. The space_kind for the rest can be handled in the implementations that need them. Otherwise no test coverage. Does that make sense to you @calixteman ?

Reply:
What's the SLOC off by 1 issue ?

Reply:
Well, I left this note for myself while documenting my changes. And no I don't generally make my notes as if I am talking to myself but I had initially thought to add to the documentation as I went.

> As you implement tests, you may find yourself running into an off by 1 error for SLOC count. 
> For most code we expect the `SpaceKind` to be `SpaceKind::Unit` but the default is `SpaceKind::Unknown`.
> To solve this you need to implement the `Getter` trait in [src/getter.rs](https://github.com/mozilla/rust-code-analysis/blob/master/src/getter.rs). Specifically, the `get_space_kind` function.

I think it is because any java snippet of code gets a `Program` leaf. I guess `UNKNOWN` gets counted and `UNIT` does not? Or maybe the other way around? It was a few months ago that I wrote it...

Reply:
@calixteman does this one make sense? I imagine the rest will be added for other metrics as needed. Possibly needed for other metrics counting functions? I have not looked at anything but LoC and the Java Cyclomatic PR.